spec/factories/comments.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+
+  factory :comment do
+    name 'Matz'
+    body 'Ipsum'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.